app/services/course/assessment/question/programming/python/python_package_service.rb

```diff
@@ -176,7 +176,7 @@ def generate_zip_file(data_files_to_keep)
 
   def zip_test_files(test_type, zip) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     tests = @test_params[:test_cases]
-    tests[test_type]&.each_with_index(1) do |test, index|
+    tests[test_type]&.each.with_index(1) do |test, index|
```

Comment:
Sorry think you are right ... should be `tests[test_type]&.each.&.with_index(1) do ...`